### ISSUE TITLE
Docker cannot download images

### ISSUE THREAD

**Issue by Eneuman** [docker/for-win#1581]
I installed Docker for Win Stable and ran "docker run hello-world"

### Expected behavior
Docker hellow world image should be downloaded

### Actual behavior
The following error message is displayed:
**docker: Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers).**

### Information

Docker installed Hyper-V
No VPN Software installed
Firewalls are all of
Windows 10 Pro 15063.850

Diagnostic ID: 7F5738A1-A330-4056-9DD2-72CE5287C3F0/2018-01-18_22-04-39

### Steps to reproduce the behavior

  1. Download latest docker for win and install it
  2. run the following command "docker run hello-world"

**Comment by msgeissler** [comment]
We get the same problem. Can you check your log if you also get these errors from vpnkit? Might be related to https://github.com/docker/for-win/issues/1238.

> vpnkit.exe: a3af Query:0 na:c:r:rn 0 <qs:registry-1.docker.io.localdomain. <A|IN>> an: au: ad: lookup failed: no servers configured

This is on a fresh installation of docker. I also cannot pull any image, so docker is basically non-operational.
I am using docker 17.12.0-ce-win47 (15139) on Windows 10 Pro Build: 16299.
I already tried all the DNS and disable IPv6 stuff, nothings works. Also already uninstalled all VPNs and VirtualBox, reinstalled docker and rebooted multiple times.
We have no proxy in our network.

**Comment by Eneuman** [comment]
Sure, how do I pull the logs ?

**Comment by msgeissler** [comment]
You can access them on the same settings page as the diagnostics:
Settigns -> Diagnose & Feedback
You can then find the log-file linked in the description.

But I believe this to be a bug in Docker now. I just switched to the (actually older) edge version of Docker and now I can pull images.

**Comment by Eneuman** [comment]
Switching to Edge didnät work for me. My logs says this:
```
[08:57:38.404][VpnKit         ][Info   ] vpnkit.exe: Connected Ethernet interface f6:16:36:bc:f9:c6

[08:57:38.404][VpnKit         ][Info   ] vpnkit.exe: UDP interface connected on 192.168.65.1

[08:57:38.404][VpnKit         ][Warning] vpnkit.exe: 4902 Query:0 na:c:r:rn 0 <qs:registry-1.docker.io. <AAAA|IN>> <an:> <au:> <ad:> lookup failed: no servers configured
```

So I guess its the same error

**Comment by rafalglowacz** [comment]
For me it was the Edge build that worked. I can finally pull images, thanks for your comment @dfatt.

**Comment by Eneuman** [comment]
Fixed in the latest Edge version 1.8
